src/app/modules/main/chat_section/module.nim

```diff
@@ -702,7 +702,7 @@ method onCommunityChannelEdited*(self: Module, chat: ChatDto) =
 method switchToOrCreateOneToOneChat*(self: Module, chatId: string) =
   # One To One chat is available only in the `Chat` section
   if(self.controller.getMySectionId() != singletonInstance.userProfile.getPubKey()):
-    return
+    self.delegate.setActiveSectionById(singletonInstance.userProfile.getPubKey())
 
   if(self.chatContentModules.hasKey(chatId)):
```

Comment:
The community cannot know if the chat exists in the chat module, so this condition will always return false in the case of community sections.